upup/pkg/fi/cloudup/openstack/cloud.go

```diff
@@ -18,25 +18,90 @@ package openstack
 
 import (
 	"fmt"
+	"time"
 
+	"github.com/golang/glog"
+	"github.com/gophercloud/gophercloud"
+	os "github.com/gophercloud/gophercloud/openstack"
+	cinder "github.com/gophercloud/gophercloud/openstack/blockstorage/v2/volumes"
 	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/kops/pkg/apis/kops"
 	"k8s.io/kops/pkg/cloudinstances"
 	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/util/pkg/vfs"
 	"k8s.io/kubernetes/federation/pkg/dnsprovider"
 )
 
+const TagNameEtcdClusterPrefix = "k8s.io/etcd/"
+const TagNameRolePrefix = "k8s.io/role/"
+const TagClusterName = "KubernetesCluster"
```

Comment:
FYI, we use this tag because that's what the AWS cloudprovider originally defined for isolating two clusters from each other.  Not sure if you have a similar concept in the openstack cloudprovider already - if not I'd recommend choosing a better value.  On AWS we're trying to move to `k8s.io/cluster/<clustername>`: https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/validation/aws.go